muttdown/main.py

```diff
@@ -137,11 +140,13 @@ def main():
 
     if args.print_message:
         print(rebuilt.as_string())
+    elif args.sendmail_passthru:
+        stdin, stdout = os.popen2(" ".join([c.sendmail, '-f', args.envelope_from, " ".join(args.addresses)]))
```

Comment:
can you change this to use `subprocess.check_call` rather than the deprecated `os.popen2`? Bonus points if you set it up with `shell=False` rather than string concatenation and passing through `/bin/sh`.